ts/a11y/explorer/KeyExplorer.ts

```diff
@@ -188,16 +188,21 @@ export class SpeechExplorer extends AbstractKeyExplorer<string> {
    * @override
    */
   public Start() {
+    let options = this.getOptions();
     if (!this.init) {
       this.init = true;
-      sreReady.then(() => {
-        this.Speech(this.walker);
-        this.Start();
-      }).catch((error: Error) => console.log(error.message));
+      sreReady().then(() => {
+        if (SRE.engineSetup().locale !== options.locale) {
+          SRE.setupEngine({locale: options.locale});
+        };
+        sreReady().then(() => {
+          this.Speech(this.walker);
+          this.Start();
+        }).catch((error: Error) => console.log(error.message));
+      });
```

Comment:
I'm wondering if the `catch()` here should be moved down to the outer `sreReady()` promise?  That way, it would catch errors not only ion the `Speech()` and `Start()` calls, but also the `setupEngine()` call.

Also, in the earlier version, the locale was checked every time the KeyExplorer's `Start()` was called, and now it is only done when `this.init` is false (so only once).  Does that matter?  Do the options ever change after initialization?

Reply:
I moved it inside, as `setupEngine` does it's own error handling. But it is a good idea to have a catch for every outer promise. 

Currently, the only way to change locale is by adding `a11y` option. We can't do it from the menu yet. So it will only happen on init. But I have a think about how it could be done more generally.